Updating time to connect for clarity (#105417)
Azure/azure-content: articles/virtual-desktop/insights-glossary.md

When monitoring time to connect, keep in mind the following things:

> - Time to connect is measured with the following checkpoints from Azure Virtual Desktop service diagnostics data. Note that the checkpoints used to denote when the connection has been established are different for a desktop versus a remote application scenario:

Reviewer: ```suggestion
- Time to connect is measured with the following checkpoints from Azure Virtual Desktop service diagnostics data. The checkpoints Insights uses to determine when the connection is established are different for a desktop versus a remote application scenario.
```

  - Begins: [WVDConnection](/azure/azure-monitor/reference/tables/wvdconnections) state = started

>   - Ends: [WVDCheckpoints](/azure/azure-monitor/reference/tables/wvdcheckpoints) Name = ShellReady (desktops); Name = RdpShellAppExecuted (RemoteApp, for timing consider the first app launch only)

Reviewer: ```suggestion
  - Ends: [WVDCheckpoints](/azure/azure-monitor/reference/tables/wvdcheckpoints) Name = ShellReady (desktops); Name = RdpShellAppExecuted (RemoteApp. For timing, consider the first app launch only)
```

>   For example, the time for a desktop experience to launch would be measured based on how long it takes to launch Windows Explorer (explorer.exe). The time for a remote application to launch would be measured based on the time taken to launch the first instance of the shell app for a connection.

Reviewer: ```suggestion
For example, Insights measures the time for a desktop experience to launch based on how long it takes to launch Windows Explorer. Insights also measures the time for a remote application to launch based on the time taken to launch the first instance of the shell app for a connection.
```

  
>[!NOTE]

> >It is possible for multiple shell app execution events to happen in the context of one connection if a user launches more than one remote application. To ensure an accurate representation of the time to connect, only the first execution checkpoint to occur in the connection should be used for a given connection.

Reviewer: ```suggestion
>If a user launches more than one remote application, sometimes the shell app can execute multiple times during a single connection. For an accurate measurement of time to connect, you should only use the first execution checkpoint for each connection.
```

> Example query:

Reviewer: ```suggestion
The following code block is an example of a kusto query used to request information about time to connect:
```

> ```kusto

Reviewer: None of the other sections have example queries. This glossary is used to define terms, not give how-to instructions for how to run queries. This information might be more suited for a how-to article like the one we use for Log Analytics at [Collect and query connection quality data](https://learn.microsoft.com/en-us/azure/virtual-desktop/connection-quality-monitoring).

Author: I'm not sure I'm aligned with that point of view since the glossary section here is specifically referencing Log Analytics table content. I think an example query is warranted for clarity.
